Sources/Sentry/SentrySession.m

```diff
@@ -186,8 +186,10 @@ - (void)changed
 - (void)incrementErrors
```

Comment:
Not sure if / how to name `incrementErrors`, since it no longer technically increases anything. But it's a good name, and I can't really think of anything better. Suggestions welcome!

Reply:
`-[SentrySession encounteredError]` would be my suggestion.

Reply:
Nice!